app/src/main/res/layout/activity_main.xml

```diff
@@ -9,6 +9,77 @@
     tools:context=".MainActivity">
 
 
+    <androidx.camera.view.PreviewView
+        android:id="@+id/viewFinder"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@+id/bottomButtonsLayout"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/topButtonsLayout">
+
+    </androidx.camera.view.PreviewView>
+
+
+    <ImageView
```

Comment:
That's unrelated, but shouldn't we put the items known to be show over the preview inside the view?

Reply:
Just tried, for somehow reason findViewById doesn't work for the elements inside the view (returns null)